### DISCUSSION TITLE
Can't generate higher resolutions after updating as of 05-29-2023

### DISCUSSION THREAD

**Discussion by ArchAngelAries** [lshqqytiger/stable-diffusion-webui-directml#144] [Q&A]
For some reason I'm no longer able to generate anything higher than 768 x 768... On my last install I could generate up to 1024 x 1536. I'm on a 7900 XT. Any help would be greatly appreciated.

my webui-user.bat settings:     

set PYTHON=
set GIT=
set VENV_DIR=
set PYTORCH_CUDA_ALLOC_CONF=garbage_collection_threshold:0.9,max_split_size_mb:512
set COMMANDLINE_ARGS=--medvram --no-half --no-half-vae --opt-split-attention --opt-sub-quad-attention --disable-nan-check --precision full --autolaunch
set SAFETENSORS_FAST_GPU=1

call webui.bat

**Comment by lshqqytiger** [comment]
It might be related: https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/10763

**Reply by songib** [reply]
@lshqqytiger 
I think this is still a problem atm since I can't generate any images after out of memory and `--lowvram` didn't help after you tried to generate bigger images and I needed to restart the WebUI afterward.  And I still can't use the Adetailer either.

Is there any solution?

**Reply by Siralob** [reply]
@songib 
Adetailer works with the latest commit `69c8fae` with `--medvram`.
I noticed Adetailer didn't work correctly even it detected face and also in-painted in the live preview. But the output was the same as input blurry face.
So, I did update Adetailer to `f18e00b`, also added `--no-half` in webui-user.bat. It works now. (deleted no-half-vae)

Thank you @Miraihi, I don't need to put --no-half-vae in command-line arguments.

~~I'm also wondering why `vladmandic/automatic` can use adetailer without these arguments in setting tab. I unchecked box related FP32 computing... idk why, webui force to use them internally?~~

**Reply by Miraihi** [reply]
Inpainting doesn't work without `--no-half` for most people, so it's to be expected that the case is the same for Adetailer.

**Comment by Miraihi** [comment]
After the latest commit I can't use ESRGAN-based upscalers at all. Throws me the "out of memory" error. Also the black squares of SCUnet return.

**Reply by Miraihi** [reply]
Update: Upscaling and img2img works again after fiddling with the new "Optimizations" settings.
Delete `--opt-split-attention --opt-sub-quad-attention` from your webui-user.bat and try something like [this](https://i.imgur.com/1Z5bdog.png).

**Reply by ArchAngelAries** [reply]
> Update: Upscaling and img2img works again after fiddling with the new "Optimizations" settings. Delete `--opt-split-attention --opt-sub-quad-attention` from your webui-user.bat and try something like [this](https://i.imgur.com/1Z5bdog.png).

Tried this and just got "RuntimeError: mat1 and mat2 must have the same dtype"

**Comment by howDoILearnPython** [comment]
then can we, the AMD card user, use the Doggettx optimization to temporarily solve the problem??

**Comment by howDoILearnPython** [comment]
tried.  i got similar VRAM usage when i select sub-quadratic in the optimizations page inside setting.
